### ISSUE TITLE
Great Data, Now what?

### ISSUE THREAD

**Issue by AydenB** [ErlerPhilipp/dr2_logger#10]
First off, Great Job. This was a breeze to install and get running. It would be great if there was a more of a "now that you have your data, here's how to improve your tunning." For example: Gear Utilization- What should you be striving for to setting up a better tune? I see that the optimal range is green, am I suppose to have the most time spent in that area? It would be nice to see this type of depth in the documentation, or even a, if you see ABC do XYZ.

**Comment by ErlerPhilipp** [comment]
Thanks, I'm glad you like it.

> Great Data, Now what?

That is THE question, also for me.

> It would be nice to see this type of depth in the documentation, or even a, if you see ABC do XYZ.

I love to give this kind of advice. Honestly, I don't think that I know enough about what an optimal setup actually is. I'm just a simulation enthusiast, not a mechanic. Things like "try to get a similar usage of all gears" is just my guess, not backed by any formal sources.
I think, there are 3 main issues that make giving such advices very difficult: (1) variance in race skills, (2) variance in car statistics and (3) driving and setup are connected.

(1): Every run is different. A single non-severe crash can probably mess up the data so badly that the optimal RPM estimation will output just bullshit. When I compare e.g. a test run (only 1st section) with a full run (all sections), there are already pretty different results. It's also hard to give useful numbers since e.g. race time and avg. speed are similar but but the acceleration varies a lot.
(2): Every car is different. The biggest differences are probably between RWD, FWD and 4WD cars. E.g. the slip behavior is so different that my full-throttle detection failed initially with RWD and FWD. A compressor also makes a huge difference, e.g. in the power-RPM plot. 
(3): Imagine you optimize the suspension so that the wheels touch the ground longer. Your car will have more grip and you will feel more secure so you drive faster. Because you drive faster, you will jump longer. In the analysis, the plots will look pretty similary, just a bit squeezed on the time axis. When you try to shift earlier or later to match the estimated optimal RPM, the collected data will be different and the program may suggest a different optimal RPM. There is a complex connection between setup and driving and this tool can't really differentiate between them.

For the readme, I only had plots of the default setup in the beginning. Then I did a lot of trial-and-error with some support from the tool. It looks like the "try to get similar usage of the gears" is true but that was just a guess. In the end, I had again some plots but didn't really know why they are different. 

In short: If you could provide some more comparisons like in the readme, I can perhaps derive some correlations and give more accurate advices. Also, I need more expert knowledge. Do you know e.g. some good tutorials, interviews, rules of thumb or similar that helps me recognize a good setup?

**Comment by ErlerPhilipp** [comment]
Maybe, it makes sense to let users upload and share their logs, plots and setups. I could perhaps make a space on my website but that would be some effort.

**Comment by ErlerPhilipp** [comment]
I added a rather simple tuning guide to the docs. I hope this answers most questions.
